orchestra/static/orchestra/timing/timecard/controller.js

```diff
@@ -9,6 +9,41 @@
       vm.orchestraTasks = orchestraTasks;
       vm.timeEntries = timeEntries;
 
+      vm.weekStart = moment().startOf('isoweek').toDate();
+      vm.weekEnd = moment().endOf('isoweek').toDate();
+
+      vm.dataLoading = true;
+      orchestraTasks.data.then(function() {
+        timeEntries.data.then(function() {
+          vm.dataLoading = false;
+
+          vm.minDate = timeEntries.minDate;
+          vm.maxDate = timeEntries.maxDate;
+          $scope.$watch(function() {
+            return [vm.minDate, vm.maxDate];
+          }, function(newVal, oldVal) {
+            if (!angular.equals(newVal, oldVal)) {
+              timeEntries.updateEntries(newVal[0], newVal[1]);
+            }
+          }, true);
+
+          timeEntries.entries.forEach(function(entry) {
+            entry.editData = vm.initialEditData(entry);
+            $scope.$watch(function() {
+              return entry.editData.date;
+            }, function(newVal, oldVal) {
+              if (newVal !== oldVal) {
```

Comment:
I think so in this case (since the object reference itself is being changed by the date picker), but `angular.equals` is safer since they're moment objects so I'll do that.